### PR TITLE
supressed the warnings by hyppo library and raised during runtime

### LINKED ISSUE #631: [BUG] giant series of runtime warnings in MCC tutorial
https://graspy.neurodata.io/tutorials/connectomics/mcc.html
see [11]

### PR DESCRIPTION
**Pull request by rajpratyush**
#### Reference Issues/PRs
closes #631

### CODE REVIEW

**Comment by loftusa** [comment]
tests are all broken?
something wrong with hyppo, apparently, might not actually be a function of this pr...

**Comment by rajpratyush** [comment]
do in need to update this PR or is it okay? @bdpedigo @loftusa

File: docs/tutorials/connectomics/mcc.ipynb [hunk 1]
```diff
@@ -273,21 +273,24 @@
...
     "    \n",
     "    # Calculate the p-value for the i-th vertex\n",
-    "    statistic, pvalue, _ = KSample(\"MGC\").test(\n",
-    "        *samples, reps=250, workers=-1\n",
-    "    )\n",
-    "    vertex_pvals.append([vertex_i, statistic, pvalue])"
+    "        statistic, pvalue, _ = KSample(\"MGC\").test(\n",
+    "            *samples, reps=250, workers=-1\n",
+    "        )\n",
+    "        vertex_pvals.append([vertex_i, statistic, pvalue])"
```
Comment (loftusa) [inline docs/tutorials/connectomics/mcc.ipynb:293]: are these tabs intentional?

Comment (rajpratyush) [reply docs/tutorials/connectomics/mcc.ipynb:293]: yes @loftusa these tabs are necessary as this particula code was giving out huge no of warnings that were runtime and by hyppo library so these had to be suppressed.
this is the modified code
`with warnings.catch_warnings():
    warnings.filterwarnings("ignore", module="hyppo")
    warnings.filterwarnings("ignore", category=RuntimeWarning)
    vertex_pvals = []

    for vertex_i in range(n_vertices):
    
    # Get the embedding of the i-th vertex for 
        samples = [
            omni_embedding[mice.labels==genotype, vertex_i, :] 
            for genotype in np.unique(mice.labels)
        ]
    
    # Calculate the p-value for the i-th vertex
        statistic, pvalue, _ = KSample("MGC").test(
            *samples, reps=250, workers=-1
        )
        vertex_pvals.append([vertex_i, statistic, pvalue])`

while this being the original code 
`vertex_pvals = []

for vertex_i in range(n_vertices):
    
    # Get the embedding of the i-th vertex for 
    samples = [
        omni_embedding[mice.labels==genotype, vertex_i, :] 
        for genotype in np.unique(mice.labels)
    ]
    
    # Calculate the p-value for the i-th vertex
    statistic, pvalue, _ = KSample("MGC").test(
        *samples, reps=250, workers=-1
    )
    vertex_pvals.append([vertex_i, statistic, pvalue])`

and as you can see the original code has been put under a function to catch the warnings and then ignore them hence adding indentation was necessary.

Comment (loftusa) [reply docs/tutorials/connectomics/mcc.ipynb:293]: @rajpratyush Oh cool, didn't see the `with` statement. Seems fine to me!

**Review by loftusa — APPROVED** [review]

**Pull request merged**